tests/test_cryo.py

```diff
@@ -0,0 +1,4 @@
+from mx3_beamline_library.devices.cryo import cryo_temperature
+
+def test_cryo_temperature():
+    assert cryo_temperature.get() == 270.0
```

Comment:
Same here - actual cryo_temp will vary, and the two most common modes we operate in are room temp and 100K. But in both cases the real measurement will fluctuate. 

Or is this comparing with a testing fixture?

Reply:
Same as above. This is just a SIM signal where the initial value is fixed